shanoir-ng-import/src/main/java/org/shanoir/ng/importer/ImporterManagerService.java

```diff
@@ -259,18 +265,24 @@ private long createRandomLong() {
 	 * @param patients
 	 * @throws ShanoirException
 	 */
-	private void downloadAndMoveDicomFilesToImportJobDir(final File importJobDir, List<Patient> patients) throws ShanoirException {
+	private void downloadAndMoveDicomFilesToImportJobDir(final File importJobDir, List<Patient> patients, ShanoirEvent event) throws ShanoirException {
 		for (Iterator<Patient> patientsIt = patients.iterator(); patientsIt.hasNext();) {
 			Patient patient = patientsIt.next();
 			List<Study> studies = patient.getStudies();
 			for (Iterator<Study> studiesIt = studies.iterator(); studiesIt.hasNext();) {
 				Study study = studiesIt.next();
 				List<Serie> series = study.getSeries();
+				int nbSeries = series.size();
```

Comment:
Hi Youenn, does the importJob for importFromPacs only contain selectedSeries (e.g. user selects in the tree 5 series out of all e.g. 10 series) as we do not verify here for "selected" on the serie? Maybe as users always take the entire exam we never realized the difference? To be checked please. Thank you!

Reply:
and this operation takes a lot of time